docs/pages/index.mdx

```diff
@@ -50,7 +50,10 @@ Teleport is a Certificate Authority and an Access Plane for your infrastructure.
     Create an issue in GitHub.
   </TileListItem>
   <TileListItem href="https://goteleport.com/get-started/">
-    Reach out sales for a demo of Teleport Enterprise.
+    Reach out to sales for a demo of Teleport Enterprise.
+  </TileListItem>
+  <TileListItem href="https://goteleport.com/signup/">
+    Try Teleport hosted for you.
```

Comment:
"hosted for you" is a bit confusing for me. Not sure if it means self-hosted or cloud.

I suspect you mean cloud, in which case you should just say "Try Teleport Cloud"

Reply:
used @ptgott 's suggestion